src/modules/cdp.ts

```diff
@@ -364,6 +365,28 @@ export class CDPModule extends BaseModule {
     }, opts)
   }
 
+  public async updateRewardScheme(params: CDPModule.UpdateRewardSchemeParams, opts?: CarbonTx.SignTxOpts) {
+    const wallet = this.getWallet()
+    const value = MsgUpdateRewardScheme.fromPartial({
+      updator: wallet.bech32Address,
+      updateRewardSchemeParams: {
+        rewardSchemeId: params.rewardSchemeId,
+        rewardDenom: params.rewardDenom,
+        rewardType: params.rewardType,
+        assetDenom: params.assetDenom,
+        rewardAmountPerSecond: params.rewardAmountPerSecond ? params.rewardAmountPerSecond.toString(10) : undefined,
+        startTime: params.startTime,
+        endTime: params.endTime,
+      }
+    })
+
+    return await wallet.sendTx({
+      typeUrl: CarbonTx.Types.MsgUpdateRewardScheme,
+      value,
+    }, opts)
+  }
```

Comment:
There is another `MsgUpdateRewardScheme` function under `src/modules/admin.ts`. Now that you've added a new one, could you delete the other one so that no one gets confused?

Also, can you check for other txs as well? (e.g. MsgCreateRewardScheme, etc.)